Nasal/physics.nas

```diff
@@ -562,6 +571,9 @@ var poll_gear_delay = func
 
 var physics_loop = func
 {
+	var nasalInit = setlistener("/sim/freeze/replay-state", func{
```

Comment:
This doesn't look right. You're creating a listener during every iteration of the loop. What I meant is something like this:

```javascript
setlistener("/sim/freeze/replay-state", func (n) {
    if (n.getBoolValue()) {
        physics_loop_timer.stop();
    }
    else {
        physics_loop_timer.start();
    }
});
```